app/views/have_read_books/index.html.erb

```diff
@@ -1,19 +1,48 @@
-<div class="sort">
-  Sort by:
-  <%= sortable "title" %>
-  <%= sortable "author" %>
-  <%= link_to "Date Completed", @have_read_books_path %>
+<div class="sort grid-x grid-padding-x grid-padding-y">
+  <div class="cell small-2 medium-2 large-1 align-self-middle">
+    <div class="text-right">Sort by</div>
+  </div>
+  <div class="links cell auto">
+    <%= sortable "title" %>
+    <%= sortable "author" %>
+    <%= link_to "Date Completed", @have_read_books_path, class: "default" %>
+  </div>
 </div>
 
-<ul class="have_read_books">
+<ul class="books-list have-read-list">
   <% @have_read_books.each do |book| %>
-    <li>
-      <%= link_to( image_tag(book.display_cover), book_path(book), class: "book-#{book.id}") %>
-      <%= book.title %>
-      <% book.authors.each do |author| %>
-        <%= author.name %>
-      <% end %>
-      <%= book.display_have_read_date(current_user) %>
+    <li class="callout secondary grid-x align-middle">
+      <div class="have-read-info cell small-12 medium-10 large-5">
+        <div class="grid-x align-middle">
+          <div class="cell small-3">
+            <%= link_to( image_tag(book.display_cover), book_path(book), class: "thumbnail-cover book-#{book.id}") %>
+          </div>
+          <div class="cell small-9 align-text-middle have-read-info">
+            <p><%= book.title %></p>
+            <p>by <%= book.display_authors %></p>
+          </div>
+        </div>
+      </div>
+      <div class="cell medium-6 large-4 have-read-rec">Not Recommended</div>
+      <div class="cell medium-6 large-3 have-read-date">
+        <% if book.display_have_read_date(current_user).nil? || params[:condition].present? %>
```

Comment:
I like that this is in a method on the Book class, but I think we should think about a way to get the controller to pre-load all of the have_read_books for a given book/user before we reach the view layer, or we'll generate 1 sql call per book.